gbpservice/nfp/config_orchestrator/common/lbv2_constants.py

```diff
@@ -0,0 +1,18 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
```

Comment:
I don't think, this file can go into common directory. Can we move these to LBV2 specific file.